packages/git/src/browser/diff/git-diff-widget.tsx

```diff
@@ -76,9 +76,9 @@ export class GitDiffWidget extends BaseWidget implements StatefulWidget {
         this.containerLayout.addWidget(this.resourceWidget);
 
         this.updateViewMode(this.scmPreferences.get('scm.defaultViewMode'));
-        this.toDispose.push(this.scmPreferences.onPreferenceChanged((e: PreferenceChangeEvent<ScmConfiguration>) => {
+        this.toDispose.push(this.scmPreferences.onPreferenceChanged(e => {
```

Comment:
Is there a reason why we no longer want to be explicit with the typing?

Reply:
It's just noise in the code base in those cases. Although things like `function (event: ...): ...` still require it.